liquidity/liquidity.go

```diff
@@ -1020,6 +1032,18 @@ func (m *Manager) currentSwapTraffic(loopOut []*loopdb.LoopOut,
 	return traffic
 }
 
+// refreshAutoloopBudget checks whether the elapsed time since our last autoloop
+// budget refresh is greater than our configured refresh period. If so, the last
+// refresh timestamp.
+func (m *Manager) refreshAutoloopBudget(ctx context.Context) {
+	if time.Since(m.cfg.AutoloopBudgetLastRefresh) >
+		m.params.AutoFeeRefreshPeriod {
+
+		log.Debug("Refreshing autoloop budget")
+		m.cfg.AutoloopBudgetLastRefresh = time.Now()
```

Comment:
nit: instead of `time.Now()` it's better to use `m.cfg.Clock.Now()` (the `clock` package is a decorator around the the usual `time` functions to allow seamless test flow).